index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+ <head>
+  <meta charset="utf-8">
+  <title>My Portfolio</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ </head>
+ <body>
+  <header>
+   <div class="icon"><p>My Logo</p></div>
+   <img src="images/Union.png" alt="">
```

Comment:
- [ ] let us try and stick to project requirements by putting our hamburger menu in `<nav></nav>` tag also kindly reduce the size of the hamburger menu as represented in the Figma design.
Figma design 👇 
    
![image](https://user-images.githubusercontent.com/66097371/148428910-29a54bad-80f1-4030-bb9d-9ebe76b6fb27.png)
your design 👇 
    
![image](https://user-images.githubusercontent.com/66097371/148429056-35b6a432-902d-4fb9-b028-510990c1cdf9.png)